libraries/chain/include/eosio/chain/block.hpp

```diff
@@ -85,21 +85,26 @@ namespace eosio { namespace chain {
 
    using block_extension = block_extension_types::block_extension_t;
 
+   using signed_block_ptr = std::shared_ptr<const signed_block>;
+   using mutable_block_ptr = std::unique_ptr<signed_block>;
```

Comment:
Why not keep the original name `mutable_signed_block_ptr`? That's more clearer.

Reply:
Because in most cases it is not "signed" yet.

Reply:
Can you add a comment? We have other cases where mutable and non-mutable use the same root like `mutale_db()` and `db()`.